.circleci/config.yml

```diff
@@ -14,7 +15,14 @@ jobs:
       - run:
           command: npm test
           name: Run tests
+      # - run: # run code coverage report
+      #     name: code-coverage
+      #     command: npm run coverage
+      # - coveralls/upload
 workflows:
   test_app:
     jobs:
       - test
+notify:
+  webhooks:
+    - url: https://coveralls.io/webhook?repo_token=b5dAPmMgZHCARyJFm6QYaOXghAF7rTD9g
```

Comment:
the token should be loaded from the .env file not exposed to the public

Reply:
done